### ISSUE TITLE
How to work with this repository?

### ISSUE THREAD

**Issue by agomezmoron** [bitnami/vms#1]
Remember by default, a search in the Issues section is done on the **opened issues**. So, how we recommend you to search in this repository before addressing a new issue:

* Firstly, look for your issues in the [resolved ones](https://github.com/bitnami/vms/issues?q=is%3Aissue+is%3Aclosed), maybe we solved similar issues before.
* If you didn't find in the resolved ones, we ask you to look on the [on going ones](https://github.com/bitnami/vms/issues). 

Finally, if you didn't find anything before, feel free to address an issue:
  * **[Technical issues](https://github.com/bitnami/vms//issues/new?assignees=&labels=help+wanted&template=technical_issues.md&title=%5BSupport%5D+Technical+issue+with):** If you have a technical issue you need support.
  *  **[How to](https://github.com/bitnami/vms/issues/new?assignees=&labels=how+to&template=how_to.md&title=%5BHow+to%5D+):** Recommended for users that needs to know how to do something in our packaged applications.
  * **[Suggestions](/issues/new?assignees=&labels=suggestion&template=suggestion.md&title=%5BSuggestion%5D+Suggestion+for):** If you want to suggest us something.
  
We will do our best to handle your request as soon as possible.

**Comment by ibushong** [comment]
Is there an archive of the old Bitnami community forum posts? I've referenced those to resolve many issues in the past, but it seems like the whole subdomain now just redirects here.

Your previous email communication said "We have migrated the most relevant content of [community.bitnami.com](http://community.bitnami.com/) to the new GitHub" but I can't find it. Where is it?

Also, fwiw, I would recommend putting up some kind of explanatory pre-redirect page on community.bitnami.com. When I'm googling for wordpress/bitnami issues, those results/urls come up **a lot**, and clicking on them just redirects to the front page of this repo, which is extremely confusing.
